src/stages/stage3_events.c

```diff
@@ -648,20 +640,19 @@ TASK_WITH_INTERFACE(scuttle_outro, BossAttack) {
 	boss->move = move_towards(VIEWPORT_W/2 - 200.0*I, 0.05);
 }
 
-TASK(scuttle_delaymove, { BoxedBoss boss } ) {
-	Boss *boss = ENT_UNBOX(ARGS.boss);
+TASK(scuttle_delaymove, { BoxedBoss boss; } ) {
+	Boss *boss = TASK_BIND(ARGS.boss);
 
-	WAIT(100);
-	boss->move.attraction_point = VIEWPORT_W/2+VIEWPORT_W/3;
-	boss->move.acceleration = 0.08;
+	boss->move.attraction_point = (VIEWPORT_W/2+200*I)+VIEWPORT_W/3;
```

Comment:
This expression can be simplified

```suggestion
	boss->move.attraction_point = 5*VIEWPORT_W/6 + 200*I;
```